apps/cb-metar/server/dataFunctions/data_map.js

```diff
@@ -110,7 +112,7 @@ dataMap = function (plotParams, plotFunction) {
 
       // send the query statement to the query function
       queryResult = matsDataQueryUtils.queryDBMapCTC(
-        cbPool,
+        cbPool, // eslint-disable-line no-undef
```

Comment:
Where is `cbPool` coming from? It's weird to me that this isn't imported or mentioned in the file. I'm wondering if this is related to a lot of the `no-undef` exceptions I'm seeing.

One way to avoid this might be to create a `/*globals ...*/` comment block. ESLint recommends it here: https://eslint.org/docs/latest/rules/no-undef

Reply:
cbPool is defined in the server main.js as a global variable. I will try the globals comment block and see if that takes care of the warning.

Reply:
Thanks! My fingers are crossed for the `globals` block. Especially if it cuts down on the number of `// eslint-disable-line no-undef` and `// eslint-disable-next-line no-undef` comments we have to have in the codebase.

Reply:
I've implemented this throughout cb-metar and it works quite well. I'll add it to the mysql apps when I get back to linting after the METAR app is complete. It is admittedly a much smaller problem in the mysql apps.